bedwars-api/src/main/java/com/andrei1058/bedwars/api/language/Messages.java

```diff
@@ -23,7 +23,6 @@
 import com.andrei1058.bedwars.api.configuration.ConfigPath;
 
 @SuppressWarnings("WeakerAccess")
-@Deprecated
```

Comment:
You cant just remove the Deprecated warning...

Reply:
I removed only the deprecated that was being given to the whole class without interfering with the messages that I would go has deprecated with the sidebar update